tests/io_test.py

```diff
@@ -238,13 +238,32 @@ def testGetDataSets(self):
         :raise: AssertionError if the input and returned dataset names don't match
         '''
         self._create_file(columns=self.dict_columns, 
-                          path_prefix='{}/iotest_dict_columns'.format(IOTest.io_test_dir))     
+                          prefix_path='{}/iotest_dict_columns'.format(IOTest.io_test_dir))     
         datasets = ak.get_datasets('{}/iotest_dict_columns_LOCALE0'.format(IOTest.io_test_dir))
 
         self.assertEqual(3, len(datasets)) 
         for dataset in datasets:
             self.assertIn(dataset, self.names)
 
+    def testStringsIO(self):
+        strings_array = ak.array(['string {}'.format(num) for num in list(range(1,11))])
+        strings_array.save('{}/strings-test'.format(IOTest.io_test_dir))
+        r_strings_array = ak.load_all('{}/strings-test'.format(IOTest.io_test_dir))['strings_array']
```

Comment:
The inverse of `Strings.save` should be `ak.load`, not `ak.load_all`. As noted above, `ak.save_all` and `ak.load_all` are inverses of each other. I'd like to see a test that has, say, 2 strings arrays, an int array, and a float array and puts them all in a dictionary, then calls `ak.save_all(my_arrays, some_path)`, and finally reads back with `my_arrays2 = ak.load_all(some_path)` and checks for equality.

Reply:
Cool, will update accordingly.

Reply:
Fixed